includes/definitions/cxr-networks.yaml

```diff
@@ -0,0 +1,12 @@
+os: cxr-networks
```

Comment:
Seems this is the vendors name. If they don't have a name for the firmware / os they run then please use something details the product line rather than just the vendor name.

Reply:
cxr-ts probably better

Reply:
cxr-ts probably better

Reply:
Shorter would be better :)

Reply:
Will do ... tomorrow ;)

Reply:
Done ;)